### ISSUE TITLE
Need to generate and upload "true mean" (MeanUCERF3) files in new modular format

### ISSUE THREAD

**Issue by ioannis-vm** [opensha/opensha#59]
Hello!

I am working in `HazardCurveGUI-2021_11_13-b736875b.jar`.
When I select Mean UCERF3 as my Earthquake Rupture Forecast, I get the following errror:

> Error downloading FM3_1_mean_ucerf3_sol.zip.
> Server down or file moved, try again later.

Hitting OK, I get

> java.io.FileNotFoundException: http://opensha.usc.edu/ftp/ucerf3_erf_modular/FM3_1_mean_ucerf3_sol.zip

After some investigation I found that file [here](http://opensha.usc.edu/trac/wiki/UCERF3FaultSystemSolutions) and I figured that it needs to be placed in a folder called `.opensha` locaed in my home directory. After doing this, it works. The same applies to `FM3_2_mean_ucerf3_sol.zip`, in case I select it in the Mean UCERF3 Presets menu.
Now, it works like that until I close the app and launch it again. In that case, when I click compute I get an error message saying "entry". When I click ok, the "Updating forecast..." status bar just hangs. It only works again after a relaunch if I manually remove the cache file `cached_FM3_1_dep100.0_depMean_rakeMean.zip` from `$HOME/.opensha`. Removing this makes it work again.
I thought I should let you know. Thank you!

---
OS: Ubuntu 20.04.3 LTS x86_64
Kernel: 5.11.0-40-generic

---

Sidenote: I also have a theoretical question, related to the required input that would reproduce the results of the [Unified Hazard Tool](https://earthquake.usgs.gov/hazards/interactive/). Should I open an issue for that or is there a more appropriate means of communication for these kinds of questions? Thank you again.

**Comment by kevinmilner** [comment]
Thanks for bringing this to my attention! We're currently transitioning to a new file format for UCERF3 & future model data, and it looks like I hadn't yet generated the MeanUCERF3 files in the new format. For now I have copied the old files over to the new location, so the download should work going forward, but I'll rename this ticket and leave it open until I regenerate all of the necessary files in the new format, which I will need to do before the next release.

I also just committed a fix for the issue that was causing you to have to delete that cache file.
